thruster_hardware/src/hardware.cpp

```diff
@@ -158,6 +182,29 @@ void ThrusterHardware::stop_thrusters()
   }
 }
 
+std::vector<std::string> ThrusterHardware::split(const std::string & str, char delimiter)
+{
+  std::vector<std::string> tokens;
+  std::stringstream ss(str);
+  std::string token;
+  while (std::getline(ss, token, delimiter)) {
+    tokens.push_back(token);
+  }
+  return tokens;
+}
+
+std::string ThrusterHardware::concatenate_strings(const std::vector<std::string> & vec)
```

Comment:
Is there anything in the STL that you can use for this? You might be able to use the views API